### ISSUE TITLE
Scheduling only priority pods while there are priority pods in the queue

### ISSUE THREAD

**Issue by netang** [volcano-sh/volcano#3832]
### Please describe your problem in detail

Is it possible to run Spark applications (on Kubernetes) using Volcano so that only high-priority pods are scheduled first and low-priority pods wait until all high-priority pods are scheduled?

When I run Spark applications using the default Kubernetes scheduler, I get situations where "small" pods with low priority are scheduled before pods with high priority (https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#effect-of-pod-priority-on-scheduling-order). So I started looking towards Volcano.

**Comment by JesseStutler** [comment]
Yes, volcano can, high-priority pods will pop from queue before the low-priority pods, but you may have to specify them in the same queue.
